### ISSUE TITLE
material-ui SelectField broken 

### ISSUE THREAD

**Issue by macrozone** [vazco/uniforms#884]
The SelectField on material-ui seems to have layout problems, its easily reproducable on the playground:

https://uniforms.tools/playground/#?N4IgDgTgpgzlAuIBcICCATd0YwAQAoBlASwFswAbKQgYwAspSBDAShABpwIB7MGZUEwCu8bjCYA3KMgBmTCnE7DR4qQBEoFJgE9kARgAMBzumLiARlXSz5ikFvObk8CEKidKTGlDrcK6KAgbBXcQaCZ0AHkAOwpdJDkQzhh6RiZkEGioAHdcEnIqWgZmACYAIQhidABzKHwAHWjcXCzc_MpqVOZ8YEbm5plubiRcYHhtMCgRwhdiaOr2XHkKbmyodAA1eTcYEYBtepAmQ8XD8xPcQ5pDgF0AXz7cO5ZGtmTfbIBJWLmoAFEIDwgglbKEmDBtNEaDFCEJzKRiIgQUkjhCoTEthQqkx4NJkXA7px4MU8SBmLjKvIQHcgA

on any other ui library or instyled, its correct: https://uniforms.tools/playground/#?N4IgDgTgpgzlAuIBcICCATd0YwAQAoBlASwFswAbKQgYwAspSBDAShABpwIB7MGZUEwCu8bjCYA3KMgBmTCnE7DR4qQBEoFJgE9kARgAMBzumLiARlXSz5ikFvObk8CEKidKTGlDrcK6KAgbBXcQaCZ0AHkAOwpdJDkQzhh6RiZkEGioAHdcEnIqWgZmACYAIQhidABzKHwAHWjcXCzc_MpqVOZ8YEbm5plubiRcYHhtMCgRwhdiaOr2XHkKbmyodAA1eTcYEYBtepAmQ8XD8xPcQ5pDgF0AXz7cO5ZGtmTfbIBJWLmoAFEIDwgglbKEmDBtNEaDFCEJzKRiIgQUkjhCoTEthQqkx4NJkXA7px4MU8SAhNEYOMrCA7kA

It also happens with a JSONSchema and an enum:

https://uniforms.tools/playground/#?N4IgDgTgpgzlAuIBcICCATd0YwAQAoApAZQHkA5YgYwAsoBbAQwEoQAacCAezBmVEYBXeFxiMAblGQAzRgBs4HISLGSAIlDmMAnsgCMABgMd0ASzEAjOVHQz5ikFoubk8CIKgcwWqlBpc5dCgIOwVPEGhGdFIAOzldJFkwjhhaBkZkEHx8ZlwAXgA-XGAAHRjcXCouGJh4XEYAK3F83BioAHdcVCb8YHq5OQBRCG4IGCRcNw82XEE4DVlBOXhxyfcoXABfZgBuMoqqmrrUuiYW0vKKydN4awmAcgwsWBh7tn2r-G0wKAeuCwaUCo8DeHwqkB4wXgplgEwuVyu0i4XDhXx-ExKIFqEFMMQA5piZlAYoJ6BMANqYxiE3CYiw0zFUTEAXU2YK270uFWgAEdBKZoOgKfdsVAEG9cPcAF6mMASkXwRjwKD3ZmciqbPYxD7SQQxYGmaqVSLKgBq8lM6CVXAg-BO6Vy8Kuh1quHEFqtIggLUa4gAdFV6GBTNY7WkmLsyuzoPBBBByvh6FwgnJcoViuyKu65JbrbakynI9quVdTNICNnc16_cFRngAGT1t0evM1kY2mB-6z4-A0R2Zq4xuPlPpBRUh1aVz02tt1rZahEa9maj5s4sHaqu-1Mc056fevLGqBKqC7qs2sOnFhaj5D-OtDq4EgUahXgBCOPQeKgl_SM23jBnvuRbbDk7BYv47QAJJxLiUDDKMoQOIwMDaPqsTEIIFj0DcSHhChaFULEQEnnhmwcL2DBSCgTDKji8ggJsQA

**Comment by wadamek65** [comment]
Hi @macrozone , thanks for the issue report. 

I've checked out the problem briefly but unfortunately was unable to find the root cause of this problem. We will have to investigate this more thoroughly.

I did find a workaround for now though. If you override the `max-height` property in the material popover menu element  it will start working as expected. For example add the following snippet in your css (note that this might affect some other components though unlikely):
```csss
.MuiPaper-root.MuiMenu-paper.MuiPopover-paper.MuiPaper-elevation8.MuiPaper-rounded {
  max-height: unset;
}
```

![image](https://user-images.githubusercontent.com/36732892/109025165-c66f9f80-76be-11eb-9669-d07e34901533.png)
